docs/reference/index-modules.asciidoc

```diff
@@ -42,11 +44,15 @@ otherwise it is written in non-compound format.
 	in order to disable it.
 
 `index.codec`::
+
+        experimental[]
         The `default` value compresses stored data with LZ4 compression, but
         this can be set to `best_compression` for a higher compression ratio,
         at the expense of slower stored fields performance.
 
 `index.shard.check_on_startup`::
+
+        experimental[]
         Should shard consistency be checked upon opening. When corruption is detected,
         it will prevent the shard from being opened.
 
```

Comment:
You need a `+` between these paras.  Currently the second para renders as literals